src/main/java/step4/domain/WinningJudgement.java

```diff
@@ -0,0 +1,35 @@
+package step4.domain;
+
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+
+public class WinningJudgement {
+    private Map winningMap;
+
+    private void initWinningMap() {
+        winningMap = new EnumMap(WinningPrice.class);
+        for (WinningPrice winningPrice : WinningPrice.values()) {
+            winningMap.put(winningPrice, 0);
+        }
+    }
+
+    public Map<WinningPrice, Integer> makeWinningMap(List<Lotto> lottoList, WinningLotto winningLotto) {
+        initWinningMap();
+        for (Lotto lotto : lottoList) {
+            putWinningMap(lotto, winningLotto);
+        }
+        winningMap.remove(WinningPrice.OTHER);
+        return winningMap;
+    }
```

Comment:
makeWinningMap 메서드가 위에 위치하고 initWinningMap 메서드가 그 아래 위치한다면
자연스럽게 물 흐르듯 읽을 수 있을 것 같아요! 코드는 위에서 아래로 이야기처럼 읽혀야 좋다고 생각합니다 :)

```suggestion
    public Map<WinningPrice, Integer> makeWinningMap(List<Lotto> lottoList, WinningLotto winningLotto) {
        initWinningMap();
        for (Lotto lotto : lottoList) {
            putWinningMap(lotto, winningLotto);
        }
        winningMap.remove(WinningPrice.OTHER);
        return winningMap;
    }
    
    private void initWinningMap() {
        winningMap = new EnumMap(WinningPrice.class);
        for (WinningPrice winningPrice : WinningPrice.values()) {
            winningMap.put(winningPrice, 0);
        }
    }
```